classes/request.class.php

```diff
@@ -120,8 +121,14 @@ public function send_request($endpoint, $requestbody, $method, $requesttype = 'g
             }
         }
 
+        $tiiurl = $this->get_apiurl();
+
+        if($requesttype === 'logging') {
```

Comment:
There needs to be a space after the `if`. The code checker would pick this up too.